src/mongo/client/dbclientcursor.cpp

```diff
@@ -92,16 +92,11 @@ void DBClientCursor::_finishConsInit() {
 }
 
 int DBClientCursor::nextBatchSize() {
```

Comment:
Is there some documentation somewhere that specifies that this is the correct interpretation? This has been a potent source of errors in the past